src/WebJobs.Script.WebHost/Security/ScriptSecretsType.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public enum ScriptSecretsType
```

Comment:
I have it pluralized as it seems to make sense, since this identifies whether you're working with the `FunctionSecrets` or the `HostSecrets`, which derive from/are `ScriptSecrets` "entity" (<EntityName>Type -> ScriptSecretsType)